ppl/archive/import.go

```diff
@@ -390,6 +391,9 @@ func importCompareFn(ark *Archive) expr.CompareFn {
 		if d > 0 {
 			cmp = 1
 		}
+		if d == 0 {
+			return bytes.Compare(a.Raw, b.Raw)
```

Comment:
Doesn't this ignore the ordering of the archive?

While you're making changes here, how about changing this function to match how `chunkLess` works: a check on the ordering to see if a & b need to be swapped, then compare timestamps if they aren't equal, and a final return statement that compares the raw bytes?

Reply:
Allow me to push back a bit: Does it matter if sorted descending or ascending? It's an arbitrary check to give us what we want: deterministic results.

If we do this here then we should probably also do this in the merger changes, but I'm not quite sure the best way to do that given that a merger can receive a generic compare function. I suppose we could do this where expressions are compiled?

Reply:
I agree that with or without depending on the order, the result will be deterministic. However, I think it will be more consistent & less surprising if it uses the ordering.

As for merge: you could update `merge.NewCompareFn`, as it takes a `reversed` parameter.

Reply:
Understood. Just to be clear, this will not provide a deterministic merge on compareFn that do not use this. Just want to verify we are ok with this for now?

Reply:
Actually I could add the deterministic check in expr.NewCompareFn.